psalm.xml

```diff
@@ -8,11 +8,11 @@
     phpVersion="8.1"
 >
     <projectFiles>
-        <directory name="lib/Doctrine/Common" />
+        <directory name="src" />
         <ignoreFiles>
             <!-- Remove ProxyGeneratorTest once Psalm supports native intersection https://github.com/vimeo/psalm/issues/6413 -->
-            <file name="lib/Doctrine/Common/Proxy/ProxyGenerator.php" />
-            <file name="tests/Doctrine/Tests/Common/Proxy/ProxyGeneratorTest.php" />
+            <file name="src/Proxy/ProxyGenerator.php" />
+            <file name="tests/Common/Proxy/ProxyGeneratorTest.php" />
```

Comment:
why are we exclude files and directories that are not covered by the project files ?